lib/fluent/plugin/kafka_plugin_util.rb

```diff
@@ -33,7 +33,7 @@ def log.formatter
       end
 
       def read_ssl_file(path)
-        return nil if path.nil?
+        return nil if path.nil? || path.respond_to?(:strip) && path.strip.empty?
```

Comment:
Why `respond_to?` is needed? I think path is nil or String.

Reply:
Its here only to verify its not an empty string and treat the following the same: `nil`, `""`,`"    "`

Reply:
@repeatedly is my response sufficient or do you ultimately believe "respond_to" is not required?

Reply:
> `"respond_to" is not required?` 

Yes. `path` is String object so no need `respond_to?` check.

Reply:
Actually, i recall why its added after re-running the tests.  This is to handle the case where the path is an array:
```
+    def test_read_ssl_file_when_non_empty_array
+        assert_equal(["a","b"], read_ssl_file(["a","b"]))
+    end
```

Reply:
I see. If so, this condition should be in next `else - end`, not here.

Reply:
Moving the check does not address the case of `" "` which is an empty string

Reply:
@repeatedly any thoughts?